WalletWasabi.Tests/UnitTests/GUI/CommandInterpreterTest.cs

```diff
@@ -0,0 +1,32 @@
+using Mono.Options;
+using System.IO;
+using WalletWasabi.Gui.CommandLine;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests
```

Comment:
The namespace should be changed to match the file path.

Reply:
Should be fixed